sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -1347,6 +1347,44 @@ public void testZetaSQLStructFieldAccessInTumble() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  @Test
+  public void testAggregateWithAndWithoutColumnRefs() {
```

Comment:
Nit: you might can include a comment like "this test is used to fix BEAM-8042", which can provide some context for readers. But it's not required.

Reply:
Great suggestions! Will add a message in a commit.